src/GovNotifyHandler.ts

```diff
@@ -0,0 +1,106 @@
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+
+import { Constants } from "./utils/Constants";
+
+import { BatchItemFailure } from "./utils/BatchItemFailure";
+import { EmailResponse } from "./models/EmailResponse";
+import { SendEmailProcessor } from "./services/SendEmailProcessor";
+import { HttpCodesEnum } from "./models/enums/HttpCodesEnum";
+import { getParameter } from "./utils/Config";
+import { AppError } from "./utils/AppError";
+import { EnvironmentVariables } from "./services/EnvironmentVariables";
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.EMAIL_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.EMAIL_LOGGER_SVC_NAME;
+
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
+
+let YOTI_PRIVATE_KEY: string;
+let GOVUKNOTIFY_API_KEY: string;
+class GovNotifyHandler implements LambdaInterface {
+	private readonly environmentVariables = new EnvironmentVariables(logger);
+
+	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
+	async handler(event: SQSEvent, context: any): Promise<any> {
+		if (event.Records.length === 1) {
+			const record: SQSRecord = event.Records[0];
+			logger.debug("Starting to process record", { record });
```

Comment:
The data in `src/events/govNotify_sqs_event.json` contains PII. I've suggesting a restriction of data passed to the logger to not cause an incident.
```suggestion
			logger.debug("Starting to process record", { record.messageId });
```